addons/mail/static/src/core/web/follower_list.js

```diff
@@ -22,13 +24,33 @@ export class FollowerList extends Component {
         super.setup();
         this.action = useService("action");
         this.store = useState(useService("mail.store"));
+        this.followerListId = local_id++;
         useVisible("load-more", (isVisible) => {
             if (isVisible) {
-                this.props.thread.loadMoreFollowers();
+                this.followerListView.loadMoreFollowers();
+            }
+        });
+
+        onWillStart(async () => {
+            await this.store.FollowerListView.insert({
+                id: this.followerListId,
+                thread_id: this.props.thread.id,
+                thread_model: this.props.thread.model,
+            });
+            await this.followerListView.get_follower();
+        });
```

Comment:
No `await` before `insert()`: insert is synchronous so that's unnecessary adding a micro-task tick for not reason.